src/analyses/is_threaded.cpp

```diff
@@ -58,30 +50,31 @@ class is_threaded_domaint:public ai_domain_baset
     ai_baset &ai,
     const namespacet &ns) final
   {
+    // assert(reachable);
+
```

Comment:
See my comment in the other pull request: more verbosity appreciated as you have a lot of information that others, e.g., myself, don't have...